client/src/partials/debtors/groups.create.html

```diff
@@ -0,0 +1,225 @@
+<div class="container-fluid">
```

Comment:
If this file is used for both `create` and `update` states, which it appears to be, `groups.edit.html` would be a better name.  Could you rename this to something along those lines?